src/colmap/ui/main_window.cc

```diff
@@ -682,6 +732,7 @@ bool MainWindow::ProjectOpen() {
       project_widget_->SetDatabasePath(*options_.database_path);
       project_widget_->SetImagePath(*options_.image_path);
       UpdateWindowTitle();
+      SetLastOpen("last_dir/project", QString::fromStdString(project_path));
```

Comment:
nit: it can be good to define a shared `const static char kLastDirProject[] = "last_dir/project";` variable and reuse the same in the multiple instances below.
```suggestion
      SetLastOpen(kLastDirProject, QString::fromStdString(project_path));
```

Reply:
Done!